core/src/layers/logging.rs

```diff
@@ -322,6 +322,47 @@ impl<A: Accessor> LayeredAccessor for LoggingAccessor<A> {
             })
     }
 
+    async fn copy(&self, from: &str, to: &str, args: OpCopy) -> Result<RpCopy> {
```

Comment:
You don't have to support all layers when adding a new operation for `Accessor`, but thank you for your effort!

Reply:
It's not meant to be done all at once. I just saw it used in the example, so I wanted to test it together by the way.